File: LiveProjects/ANDROID/AndroidAppComponents/Activities/Geeks4Geeks_MultiScreen_App/app/src/main/java/com/example/geeks4geeks_multiscreen_app/MainActivity.java
package com.example.geeks4geeks_multiscreen_app;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    // define the global variable

    TextView question1;
    // Add button Move to Activity

    Button next_Activity_button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // by ID we can use each component which id is assign in xml file
        // use findViewById() to get the Button
        next_Activity_button = (Button) findViewById(R.id.first_activity_button);
        question1 = (TextView) findViewById(R.id.question1_id);

        // In question1 get the TextView use by findViewById()
        // In TextView set question Answer for message
        question1.setText("Q 1 - How to pass the data between activities in Android?\n"
                + "\n"
                + "Ans- Intent");

        // Add_button add clicklistener
        next_Activity_button.setOnClickListener(new View.OnClickListener() {

            public void onClick(View v)
            {

                // Intents are objects of the android.content.Intent type. Your code can send them
                // to the Android system defining the components you are targeting.
                // Intent to start an activity called SecondActivity with the following code:

                Intent intent = new Intent(MainActivity.this, SecondActivity.class);

                // start the activity connect to the specified class
                startActivity(intent);
            }
        });
    }
}

File: LiveProjects/ANDROID/Android_FullProjects/HeadFirstAndroidProgramming/HeadfirstMy1stApplication/app/src/main/java/com/example/headfirstmy1stapplication/MainActivity.kt
//This is the package name.
package com.example.headfirstmy1stapplication


//These are Android classes used in MainActivity.
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

//MainActivity extends the Android class android.app.Activity.
class MainActivity : AppCompatActivity() {

//Implement the onCreate() method from the Activity class. This method is called
//when the activity is first created.
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

// Specify which layout to use.
        setContentView(R.layout.activity_main)
    }
}